ci/scripts/create-and-update-db.sh

```diff
@@ -34,8 +34,9 @@ for db in ${DATABASES}; do
           seed varchar(36),
           backup_code varchar(36)
         );
-      ALTER TABLE IF EXISTS totp_seed
-        DROP CONSTRAINT username_record_keeper;
+      IF EXISTS (SELECT 1 from sys.objects where name = 'username_record_keeper')
+        ALTER TABLE IF EXISTS totp_seed
+          DROP CONSTRAINT username_record_keeper;
```

Comment:
I noticed that this was failing before because it couldn't drop a constraint that didn't exist.